data/src/main/java/io/gnosis/data/repositories/TransactionRepository.kt

```diff
@@ -0,0 +1,191 @@
+package io.gnosis.data.repositories
+
+import io.gnosis.data.backend.TransactionServiceApi
+import io.gnosis.data.backend.dto.ServiceTokenInfo
+import io.gnosis.data.models.*
+import io.gnosis.data.utils.formatBackendDate
+import pm.gnosis.crypto.utils.asEthereumAddressChecksumString
+import pm.gnosis.model.Solidity
+import pm.gnosis.utils.asEthereumAddress
+import pm.gnosis.utils.decimalAsBigInteger
+import pm.gnosis.utils.hexToByteArray
+import pm.gnosis.utils.removeHexPrefix
+import java.math.BigInteger
+
+//TODO Remove all this, when we have ContractInfo from the backend
+private val defaultErc20Address = "0xc778417e063141139fce010982780140aa0cd5ab".asEthereumAddress()!!
+private val defaultErc721Address = "0xB3775fB83F7D12A36E0475aBdD1FCA35c091efBe".asEthereumAddress()!!
+private val ETH_ADDRESS = Solidity.Address(BigInteger.ZERO)
+val ETH_TOKEN_INFO = Erc20Token(ETH_ADDRESS, "Ether", "ETH", 18, "local::ethereum")
+val ETH_SERVICE_TOKEN_INFO = ServiceTokenInfo(ETH_ADDRESS, 18, "ETH", "Ether", "local::ethereum")
+val FAKE_ERC20_TOKEN_INFO = ServiceTokenInfo(defaultErc20Address, 18, "WETH", "Wrapped Ether", "local::ethereum")
+val FAKE_ERC721_TOKEN_INFO = ServiceTokenInfo(defaultErc721Address, 18, "DRK", "Dirk", "local::ethereum")
```

Comment:
To be able to check for it in a test it needs be here. The code tested does not depend on the test. So if this is only in the test, then it can't be used in the SUT.